dapr/proto/common/v1/common_pb2_grpc.py

```diff
@@ -1,3 +1,4 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
```

Comment:
@karishma-chawla was this automatically added?

Reply:
I ran the script in the repo for generated code to let it pick latest changes https://github.com/dapr/python-sdk/blob/master/tools/regen_grpcclient.sh